README.md

```diff
@@ -46,12 +46,20 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
+* Install Docker Community Edition and docker-compose
```

Comment:
FWIW the semantics of the web kind of dictate this _should_ be a numbered list. There's `ol` and `ul` (ordered and unordered list), and this is clearly something with an order. So :+1: for a list of install steps being ordered. Examples of things we should leave unordered are e.g. a list of requirements, methods of contacting us, etc.